### PR TITLE
Add check for vspec file path

### PR DESCRIPTION
**Pull request by dennismeister93**
Add check for vspec file path in vehicle databroker script, create configmap, mount to k8s deployment

Signed-off-by: Dennis Meister <dennis.meister@bosch.com>

### CODE REVIEW

File: src/runtime/deploy_runtime.sh [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 if ! helm status vehicleappruntime &> /dev/null
 then
+
```
Comment (doosuu) [inline src/runtime/deploy_runtime.sh]: Unless this was introduced by some new bash formatter we are going to use now, please remove the added whitespace/newline

**Review by doosuu — DISMISSED** [review]

**Comment by doosuu** [comment]
@dennismeister93 can you please add a pre-commit workflow and a pre-commit config.

**Review by doosuu — COMMENTED** [review]
LGTM, but I still need to perform a local test

**Review by MP91 — APPROVED** [review]
LGTM

**Pull request merged**